src/components/GrowlNotification/index.js

```diff
@@ -55,65 +43,73 @@ class GrowlNotification extends Component {
      * @param {String} type
      * @param {Number} duration
      */
-    show(bodyText, type, duration) {
-        this.setState(
-            {
-                bodyText,
-                type,
-            },
-            () => {
-                this.fling(0);
-                setTimeout(() => {
-                    this.fling(INACTIVE_POSITION_Y);
-                }, duration);
-            },
-        );
-    }
+    // eslint-disable-next-line no-unused-vars
+    const show = useCallback((text, growlType, growlDuration) => {
```

Comment:
I've re-implemented the `show` method, but was not used anywhere in the App (as well as the component, as I can see) - I think it's worth to keep it because it may be useful once the component become used again

Reply:
Ah... Thanks for pointing this out! I'll fix it then.

Reply:
resolved

Reply:
No child component is wrapped with memo, so useCallback won't do any optimization between renders